server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -478,7 +497,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
             this.sessionMetric,
             this.sequenceNumber,
             this.durableSequenceNumber,
-            this.nackMessages?.identifier,
+            this.nackMessages.size > 0 ? Array.from(this.nackMessages.keys())[0] : undefined,
```

Comment:
`logCommonSessionEndMetrics` last argument is `timeoutErrorReason`. If I understand it correctly, we are passing the first "identifier" in the `nackMessages` map as the reason. Could it be possible, if `nackMessages` size is > 1, that there is more than one `timeoutErrorReason`? Or maybe that the reason be something other than the element at position 0 of the array created above?

Reply:
I fixed this up. It will pass a list now